### PR TITLE
HamburgerMenuItem ToolTip Implementation

### PR DESCRIPTION
**Pull request by amkuchta**
## What changed?

_Creates / Assigns the `ToolTip` `DependencyProperty` for `HamburgerMenuItem`, which will then carry over into `HamburgerMenuGlyphItem`, `HamburgerMenuIconItem`, etc._

## Closed Issues
_#3169_

### CODE REVIEW

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs [hunk 1]
```diff
@@ -162,6 +167,22 @@ public bool IsEnabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that specifies label to display.
```
Comment (punker76) [inline src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: `Gets or sets a value that specifies tooltip to display.`

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs [hunk 2]
```diff
@@ -44,6 +44,11 @@ public class HamburgerMenuItem : Freezable, ICommandSource
         /// </summary>
         public static readonly DependencyProperty IsEnabledProperty = DependencyProperty.Register(nameof(IsEnabled), typeof(bool), typeof(HamburgerMenuItem), new PropertyMetadata(true, null, IsEnabledCoerceValueCallback));
 
+        /// <summary>
+        /// Identifies the <see cref="ToolTip"/> dependency property. 
+        /// </summary>
+        public static readonly DependencyProperty ToolTipProperty = DependencyProperty.Register(nameof(ToolTip), typeof(string), typeof(HamburgerMenuItem), new PropertyMetadata(null));
```
Comment (punker76) [inline src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: should be `typeof(object)`

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs [hunk 3]
```diff
@@ -162,6 +167,22 @@ public bool IsEnabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that specifies label to display.
+        /// </summary>
+        public string ToolTip
```
Comment (punker76) [inline src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: should be typeof(object)

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs (hunk 2 above)
Comment (amkuchta) [reply src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: Completed in latest commit

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs (hunk 1 above)
Comment (amkuchta) [reply src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: Completed in latest commit

File: src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs (hunk 3 above)
Comment (amkuchta) [reply src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs]: Completed in latest commit

**Comment by amkuchta** [comment]
@punker76 back to you!

**Pull request merged**